.github/actions/setup-backend/action.yml

```diff
@@ -24,6 +24,7 @@ runs:
         cache: ${{ inputs.cache }}
     - name: Install dependencies
       run: |
+        sudo apt-get update && sudo apt-get -y install libldap2-dev libsasl2-dev
```

Comment:
I think it makes sense to bring python-related os dependencies in this `setup-backend` reusable action instead of within the `bashlib.sh` that I still don't understand how it gets discovered and executed.